resources/mappings.json

```diff
@@ -1,33 +1,31 @@
 {
-   "doc":{
-      "fieldFormatMap": "{
-         \"Duration\": {
-            \"id\":\"number\",
-               \"params\": {
-                  \"pattern\":\"00:00:00\"
-               }
-         },
-         \"AttachSize\": {
-            \"id\": \"bytes\"
-         },
-         \"TotalBytesDelta\": {
-            \"id\": \"bytes\"
-         },
-         \"TotalBytes\": {
-            \"id\":\"bytes\"
-         },
-         \"SrcBytes\": {
-            \"id\":\"bytes\"
-         },
-         \"SrcBytesDelta\": {
-            \"id\":\"bytes\"
-         },
-         \"DestBytes\": {
-            \"id\":\"bytes\"
-         },
-         \"DestBytesDelta\": {
-            \"id\":\"bytes\"
-         }
-      }"
-   }
+	"fieldFormatMap": {
+		"Duration": {
```

Comment:
please explain this difference. The only change I see is the "doc", line 2 is removed and that the "escape" ```\``` of strings is removed. 

Why? How is this different?